test/helpers/web3EventObserver.js

```diff
@@ -0,0 +1,17 @@
+/*
+Copyright: Ambrosus Technologies GmbH
+Email: tech@ambrosus.com
+
+This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+This Source Code Form is “Incompatible With Secondary Licenses”, as defined by the Mozilla Public License, v. 2.0.
+*/
+
+const observeEventEmission = async (web3, codeBlock, contract, eventName) => {
```

Comment:
Don't like the word observe, for me it implies some kind of subscription or something

Reply:
This makes it consistent with `web3BalanceObserver` and `observeBalanceChange`.
If you have better idea, then plz come up with one.

Reply:
`captureEventEmission`?